esterni/verbali/verbale-esterno_2020-03-11.tex

```diff
@@ -0,0 +1,139 @@
+\end{description}
+
+\subsection{Membri del gruppo presenti}%
+\label{sub:membri_del_gruppo_presenti}
+
+\begin{enumerate}
+  \item Riccardo Agatea
+  \item Tobia Apolloni
+  \item Riccardo Cestaro
+  \item Alberto Cocco
+  \item Luca Ercole
+  \item Alberto Gobbo
+  \item Alessandro Rizzo
+  \item Fabio Scettro
+\end{enumerate}
+
+% sub:membri_del_gruppo_presenti (end)
+
+\subsection{Altri partecipanti}%
+\label{sub:altri_partecipanti}
+
+\begin{enumerate}
+  \item Davide Zanetti (Imola Informatica, proponente del capitolato)
+  \item Andrea Sabbioni (Imola Informatica, referente per il server)
+\end{enumerate}
+
+% sub:altri_partecipanti (end)
+% sec:informazioni_logistiche (end)
+
+\section{Introduzione}%
+\label{sec:introduzione}
+
+L'incontro è avvenuto tramite chat su un gruppo Telegram dedicato.
+
+Lo scopo principale era discutere della possibilità per il gruppo di utilizzare l'infrastruttura server di \glossario{Imola Informatica}.
+
+\section{Ordine del giorno}%
+\label{sec:ordine_del_giorno}
+
+\begin{itemize}
+  \item Conversazione Telegram
+\end{itemize}
+
+\section{Conversazione Telegram}%
+\label{sec:conversazione_telegram}
+
+Riportiamo per completezza la conversazione avvenuta su Telegram:
+
+Andrea Sabbioni, tecnico di Imola Informatica:
+\begin{enumerate}
+
+\item Ciao ragazzi, vi va di spiegarmi un attimo che risorse vi servono e cosa ci andate a fare sopra così cerco di darvi qualche consiglio. Tenete conto che le risorse che vi possiamo dare non hanno garanzie di \glossario{SLA} quindi cerchiamo di pianificare bene le cose
+\end{enumerate}
+
+A cui il gruppo ha risposto:
+
+\begin{enumerate}
+  \item Ciao! Siamo ancora agli inizi per cui non farti problemi a dirci che le nostre idee sono insensate.
+
+  Avevamo pensato di utilizzare le vostre risorse come dev environment remoto per il nostro server \glossario{Spring Boot} che espone un'API \glossario{RESTful}. Quindi ci basterebbe fare ssh sul vostro server e:
+
+  git clone —depth=1 repo-nostro-server
+
+  ./mvnw compile jib:build \ldots %chktex 26
+
+  docker run something
+
+  e poi fare richieste HTTP dai nostri pc usando prima \glossario{curl} e più avanti la web app e app che svilupperemo, in modo da verificare se le cose funzionano come dovrebbero.
+  Una delle prime cose che dobbiamo fare (appena capiamo come farlo) è il meccanismo di autenticazione/autorizzazione dell'API (dato che abbiamo tipologie diverse di utenti che hanno accesso a funzionalità diverse).
+
+  Abbiamo scoperto questo tool qui che dovrebbe semplificarci build e deploy dell'immagine: https://github.com/GoogleContainerTools/jib. Lo conoscevi? È qualcosa che viene utilizzato in pratica?  %chktex 44
+  Alternativamente possiamo provare a pubblicare l'immagine o su docker hub o sul GitHub package registry in modo da poter fare direttamente un docker pull
+
+  Infine: tutto questo setup ti sembra ragionevole nell'ottica di (eventualmente) usare Kubernetes per far scalare il server?
```

Comment:
Primo riferimento alla parola "Kubernetes" nel documento. In seguito viene riferita a glossario, ma è meglio farlo qui